components/alibi-detect-server/adserver/cd_model.py

```diff
@@ -37,18 +37,40 @@ def _drift_to_metrics(drift):
             {"key": "seldon_metric_drift_is_drift", "value": is_drift, "type": "GAUGE",}
         )
 
+    distance = drift.get("distance")
```

Comment:
*distance* type can either be **float** or **np.ndarray**, so proper type needs to be picked up and logged. Iterate through array and append or add float.